recipe/meta.yaml

```diff
@@ -7,34 +7,33 @@ package:
 source:
   fn: libcmaes-{{ version }}.tar.gz
   url: https://github.com/beniz/libcmaes/archive/{{ version }}.tar.gz
-  sha256: 899fcc44b551e4976a77e48e41bdd5dafc3394bd3fcb3c55996c02e4267bfa28
+  sha256: 1b23e53eb4acb123348e7d00dfdd7c7699c513950347fcde7425376ad333f6c5
   patches:
-    # https://github.com/beniz/libcmaes/pull/149
-    - py3k.patch
+    - dynamic_lookup.patch  # [osx]
 
```

Comment:
the logic should be: if windows -> link to python libs else if osx -> undefined lookup (on linux no need to do anything)
their lnk_python flag can be removed
this patch should be upstreamed then